compiler/resolution/functionResolution.cpp

```diff
@@ -1664,27 +1825,55 @@ handleSymExprInExpandVarArgs(FnSymbol*  workingFn,
         newFormal->cname        = astr("_e", istr(i), "_", formal->cname);
 
         tupleCall->insertAtTail(new SymExpr(newFormal));
-
         formal->defPoint->insertBefore(newArgDef);
+        varargFormals[i] = newFormal;
       }
 
+      bool needTupleInBody;
+
       // Replace mappings to the old formal with mappings to the new variable.
       if (workingFn->hasFlag(FLAG_PARTIAL_COPY)) {
+        bool gotFormal = false; // for assertion only
```

Comment:
Given that this is pre-existing code, do you insist on fixing the issue that you point out?